package.json

```diff
@@ -17,24 +17,25 @@
     "docs:dev": "yarn sync:NotFound; yarn sync:VPFlyout; yarn sync:VPNavBarTitle; yarn sync:VPNavBar; vitepress dev docs",
     "docs:build": "yarn sync:NotFound; yarn sync:VPFlyout; yarn sync:VPNavBarTitle; yarn sync:VPNavBar; vitepress build docs;",
     "docs:serve": "vitepress serve docs --port 8082",
-    "format": "prettier --write --cache \"./**/*.{js,vue,md,json,yaml}\" --loglevel silent",
+    "format": "prettier --write --cache \"./**/*.{js,vue,md,json,yaml}\" --log-level silent",
     "format:check": "prettier --check --cache \"./**/*.{js,vue,md,json,yaml}\"",
     "prepare": "husky",
     "flex:build": "node libs/flexBuildIndexes.js; yarn format;",
     "flex:test": "node libs/flexTestSearch.js;",
     "axios:build": "node libs/axiosBuildScripts.js; yarn format;",
     "zip:create": "node libs/createZipFiles.js;",
-    "zip:config-check": "node libs/checkZipConfigs.js;"
+    "zip:config-check": "node libs/checkZipConfigs.js;",
+    "firebase:emulator": "yarn docs:build; yarn flex:build; yarn flex:test; firebase emulators:start"
```

Comment:
I tried and got:
```
/bin/sh: 1: firebase: not found
error Command failed with exit code 127.
```

Which means for this to work `firebase` would need to be some sort of devDependency or already happen to be installed and available in one's path (which I doubt most  devs would have).

I'm a little confused what this offers beyond the PR preview and relative to `docs:dev` and `docs:serve`?

Reply:
Yes a dev would need to install the firebase CLI. However this script is not needed to work on the docs. Its main purpose is to help trouble shoot firebase issues locally. Having to do a repo preview while debugging this PR was painful. So it is just another tool that for the most part is not used that often.

Reply:
Ok- it's probably fine to leave it- I doubt anyone will use it serendipitously. Maybe just change the /dev section on it to say that the firebase CLI is necessary and otherwise for the typical developer the firebase PR preview is a simpler option?